sql/mob_droplist.sql

```diff
@@ -12883,16 +12883,22 @@ INSERT INTO `mob_droplist` VALUES (2472,0,0,1000,3225,30);
 INSERT INTO `mob_droplist` VALUES (2472,0,0,1000,18892,100);
 INSERT INTO `mob_droplist` VALUES (2472,0,0,1000,19187,100);
 INSERT INTO `mob_droplist` VALUES (2473,0,0,1000,2222,80);
-INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,2160,80);
-INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,2223,80);
+INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,2160,28);
+INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,2223,74);
+INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,2333,213);
+INSERT INTO `mob_droplist` VALUES (2474,0,0,1000,8801,73);
```

Comment:
This comment applies to all mob drops. They should conform to the chart I linked in your other PR:
https://github.com/LandSandBoat/server/pull/1129

Reply:
Got it. For those that do not have a label however, is there a standard that should be used then?

Reply:
> so compared with other drops from other mobs in same zone to reflect the same 

no, bad newbie :newspaper_roll: stop that and wait till ppl have time to answer the question